friendship-quiz.js

```diff
@@ -12,14 +12,23 @@ window.onload = init;
 
 var answerList = []; //initialize answerList to be empty
 
-/* ADD YOUR CODE BELOW */
-
 //adds user-submitted answer to answerList
 function submitAnswer() {
-
+  var answer = document.getElementById("answerInput").value;
+  document.getElementById("answerInput").value = '';
+  answerList.push(answer);
 }
 
 //checks if user-submitted guess is in answerList
 function checkGuess() {
+  var guess = document.getElementById("guessInput").value;
+  var x = answerList.length;
+  for(var i = 0; i < x; i++){
+      if(guess == answerList[i]){
```

Comment:
Minor nitpick, but the general convention is to leave a space between the closing `)` for the condition and the opening `{` for the body of the loop/`if` statement.